### DISCUSSION TITLE
arbitrary box-shadow in jit mode?

### DISCUSSION THREAD

**Discussion by littlee** [tailwindlabs/tailwindcss#4818] [Help]
I'm trying to use an arbitrary value of box-shadow, but I can not figure out the right way?

Does tailwindcss support this yet?

```html
<div class="shadow-[0px 1px 2px 0px rgba(0, 0, 0, 0.04)]"></div>
```

**Comment by dance2die** [comment]
The limitation is that classes are separated by spaces.  

You can workaround by overriding the Tailwind CSS variable, `--tw-shadow`.  
(found in [Box Shadow documentation](https://tailwindcss.com/docs/box-shadow)),

**Demo**: https://play.tailwindcss.com/WNPi4hou63?layout=horizontal
```html
<div 
    style="--tw-shadow: 0px 1px 2px 0px rgba(0, 0, 0, 1.04)" 
    class="shadow-2xl w-20 h-20 p-16 flex justify-center items-center">
  box
</div>
```

**Note**: You can use any `shadow*` utilities other than `shadow-2xl`.

**Reply by littlee** [reply]
🐮 🍺
